app/gui/view/graph-editor/src/component/node/action_bar/icon.rs

```diff
@@ -208,3 +195,47 @@ pub mod disable_reevaluation {
         }
     }
 }
+
+/// Icon for the button to enable re-evaluation. Looks like an arrow loop.
+pub mod enable_reevaluation {
+    use super::*;
+
+    ensogl::shape! {
+        (style: Style, color_rgba: Vector4<f32>) {
+            let fill_color = Var::<color::Rgba>::from(color_rgba);
+            let width = Var::<Pixels>::from("input_size.x");
+            let height = Var::<Pixels>::from("input_size.y");
+
+            let unit = &width / 16.0;
+
+            let arrow_loop = arrow_loop(&unit)
+                .fill(fill_color);
```

Comment:
why this is split into multiple lines? It will fit one line and still take <100 chars, right?